### PR TITLE
Basic Concepts Assignments

### PR DESCRIPTION
**Pull request by neo1691**
This lesson asks the students to create three variables `x`, `y`
and `z` with an int, float and string value respectively

Let me know if there is anything wrong. Like a terminology that I missed or something similar. 
Thanks!

### CODE REVIEW

File: unit-2-basic-concepts/lesson-2-variable-assignment/README.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# Variable-assignment
+
+Your job is to 
+1. Assign an interger to the variable `x` with the value `10`
```
Comment (rosiecakes) [inline unit-2-basic-concepts/lesson-2-variable-assignment/README.md]: Hey there! With markdown you want to put two spaces at the end of a sentence to create a new line, or edit in Github and use the little list button up there ^^

Comment (rosiecakes) [inline unit-2-basic-concepts/lesson-2-variable-assignment/README.md]: & typo on integer

File: unit-2-basic-concepts/lesson-4-fun-with-strings/README.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
+# Fun-with-strings
+
+Your task is to write a single line of code, that repeats the input string `s` 5 times. 
+
+You are not required to know the concepts of `function` in python. All you need to know that the string that you have to work on is stored in a variable `s`. In your solution, you need to delete the comment next to the `return` statment and write your code there. 
+
+**Note: It's important that `return` followed by your code are in one line**.
+
+*Hint*: The solution to this assignment is a single line of code.
```
Comment (rosiecakes) [inline unit-2-basic-concepts/lesson-4-fun-with-strings/README.md]: I appreciate the clarity but imo this is redundant

File: unit-2-basic-concepts/lesson-3-equation-solver-with-python/.rmotr [hunk 3]
```diff
@@ -0,0 +1,3 @@
+type = "assignment"
+uuid = "42f082ec-21be-41d5-82cd-8ab705435891"
+name = "Equation-solver-with-python"
```
Comment (santiagobasulto) [inline unit-2-basic-concepts/lesson-3-equation-solver-with-python/.rmotr]: You can safely remove the dashes: "Equation solver with python"

File: unit-2-basic-concepts/lesson-3-equation-solver-with-python/README.md [hunk 4]
```diff
@@ -0,0 +1,12 @@
+# Equation-solver-with-python
+
+## The slope equation. 
+
+Let's say you have a line with two points (x1, y1) and (x2, y2). Your task is to find the slope `m` of the line whose points are (2, 3), (6, 7). The formula to calculate the slope of theline is: 
+![img](https://www.dropbox.com/s/iw5hio9af42vual/Screenshot%202016-09-30%2008.04.41.png?dl=0)
```
Comment (santiagobasulto) [inline unit-2-basic-concepts/lesson-3-equation-solver-with-python/README.md]: The dropbox link doesn't return the actual image: https://github.com/neo1691/itp-class/blob/524cc810bb6fc911d4f0ee2b088a3e05b0102f6f/unit-2-basic-concepts/lesson-3-equation-solver-with-python/README.md

File: unit-2-basic-concepts/lesson-2-variable-assignment/README.md (hunk 1 above)
Comment (neo1691) [inline unit-2-basic-concepts/lesson-2-variable-assignment/README.md]: I really need a good markdown plugin for vim. I made the changes, hope they are correct.

File: unit-2-basic-concepts/lesson-4-fun-with-strings/README.md (hunk 2 above)
Comment (neo1691) [inline unit-2-basic-concepts/lesson-4-fun-with-strings/README.md]: I am assuming the kid's don't have absolutely any knowledge about functions and return statement.

**Pull request merged**